### PR TITLE
Simulation

### PR DESCRIPTION
**Pull request by pophaax**

### CODE REVIEW

File: simu_run.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+SIMU_PATH=/home/sailbot/sailing_simulator/PROGSIDE
```
Comment (JSpringett) [inline simu_run.sh]: Make as input, or relative path, or environmental varaible

**Comment by JSpringett** [comment]
README? How do I use this??

File: simu_run.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```
Comment (JSpringett) [inline simu_run.sh]: Relies on another repository maybe put this script in tat? OR use a git submodule

**Comment by OliverNyholm** [comment]
Good boy

**Comment by sailingrobots** [comment]
I'm John McCain and I approve this message.

(Seriously though, this is Emil)

Edit: clarification

**Comment by JSpringett** [comment]
All good

File: main.cpp [hunk 3]
```diff
@@ -64,7 +64,12 @@ int main(int argc, char *argv[]) {
 		path = "./";
```
Comment (simchanu29) [inline main.cpp:64]: Please add a quick summary of the purpose of this main and the process occurring inside.

Comment (simchanu29) [inline main.cpp:64]: Could you reformat the code to help the reading of the functions ? They sometimes are on 2 lines.
I think it should be better to put the arguments on a second line.

**Pull request merged**